nano/nano_node/daemon.cpp

```diff
@@ -125,10 +125,12 @@ void nano_daemon::daemon::run (boost::filesystem::path const & data_path, nano::
 				node->flags = flags;
 				node->start ();
 				std::unique_ptr<nano::rpc> rpc = get_rpc (io_ctx, *node, config.rpc);
-				if (rpc && config.rpc_enable)
+				if (rpc)
 				{
-					rpc->start ();
+					rpc->start (config.rpc_enable);
 				}
+				nano::ipc::ipc_server ipc (*node, *rpc);
+				config.node.disable_lazy_bootstrap = disable_lazy_bootstrap;
```

Comment:
Is this part of this changeset or is it from a merge error ?

Reply:
It should look like

```
				nano::ipc::ipc_server ipc (*node, *rpc);
				runner = std::make_unique<rai::thread_runner> (service, node->config.io_threads);
				runner->join ();
```